tcf_website/templates/landing/landing.html

```diff
@@ -1,23 +1,297 @@
+        <form>
+          <div class="form-row">
+            <div class="col-md-6">
+              <div class="form-row">
+                <div class="form-group col-md-6">
+                  <label for="inputFname">First Name</label>
+                  <input type="text" class="form-control" id="inputFname" placeholder="Karen">
+                </div>
+                <div class="form-group col-md-6">
+                  <label for="inputLname">Last Name</label>
+                  <input type="text" class="form-control" id="inputLname" placeholder="Doe">
+                </div>
+              </div>
+              <div class="form-group">
+                <label for="inputEmail">Email</label>
+                <input type="email" class="form-control" id="inputEmail" placeholder="youremail@virginia.edu">
+              </div>
+            </div>
+            <div class="col-md-6">
+              <div class="form-group">
+                <label for="inputTitle">Title</label>
+                <input type="text" class="form-control" id="inputTitle" placeholder="Message Subject">
+              </div>
+              <div class="form-group">
+                <label for="inputMessage">Feedback</label>
+                <textarea type="text" class="form-control" id="inputMessage" rows="6" placeholder="Write your message here..."></textarea>
+              </div>
+            </div>
+          </div>
+          <div class="text-right">
+            <button type="submit" class="btn btn-primary bg-indigo">Submit Feedback</button>
+          </div>
+        </form>
+      </div>
+    </div>
+  </div>
+</section>
+
+
+<!-- Testimonials (Add later with real feedback?)-->
+<!-- <section class="testimonials text-center bg-light">
+  <div class="container">
+    <h2 class="mb-5">What people are saying...</h2>
+    <div class="row">
+      <div class="col-lg-4">
+        <div class="testimonial-item mx-auto mb-5 mb-lg-0">
+          <img class="img-fluid rounded-circle mb-3" src="img/testimonials-1.jpg" alt="">
+          <h5>Margaret E., CLAS '19</h5>
+          <p class="font-weight-light mb-0">"This is fantastic! Thanks so much guys!"</p>
+        </div>
+      </div>
+      <div class="col-lg-4">
+        <div class="testimonial-item mx-auto mb-5 mb-lg-0">
+          <img class="img-fluid rounded-circle mb-3" src="img/testimonials-2.jpg" alt="">
+          <h5>Fred S., SEAS '21</h5>
+          <p class="font-weight-light mb-0">"theCourseForum is amazing. I've been using it to choose all my classes."</p>
+        </div>
+      </div>
+      <div class="col-lg-4">
+        <div class="testimonial-item mx-auto mb-5 mb-lg-0">
+          <img class="img-fluid rounded-circle mb-3" src="img/testimonials-3.jpg" alt="">
+          <h5>Sarah W., CLAS '22</h5>
+          <p class="font-weight-light mb-0">"Thanks so much for making these reviews available to us!"</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section> -->
```

Comment:
Will need real testimonials at some point

Reply:
Removed them for now